Godeps/Godeps.json

```diff
@@ -373,12 +393,12 @@
 		},
 		{
 			"ImportPath": "github.com/elazarl/go-bindata-assetfs",
-			"Rev": "3dcc96556217539f50599357fb481ac0dc7439b9"
+			"Rev": "c57a80f1ab2ad67bafa83f5fd0b4c2ecbd253dd5"
```

Comment:
We want our version, not kube's. https://github.com/kubernetes/kubernetes/pull/13835 Is open to bring kube up to our level